### PR TITLE
Added the crucial info about it being sorted up front [doc only]

### PR DESCRIPTION
**Pull request by mark-summerfield**
Plus mentioned more familiar names (e.g., if people search for it).

### CODE REVIEW

File: lib/pure/collections/critbits.nim [hunk 1]
```diff
@@ -8,8 +8,8 @@
 #
 
 ## This module implements a `crit bit tree`:idx: which is an efficient
-## container for a set or a mapping of strings. Based on the excellent paper
-## by Adam Langley.
+## container for a sorted set of strings, or for a sorted mapping of strings. Based on the excellent paper
+## by Adam Langley. (A crit bit tree is a form of radix tree or patricia trie.)
```
Comment (Araq) [inline lib/pure/collections/critbits.nim]:
```
`radix tree`:idx: or `patricia trie`:idx:
```

**Comment by mark-summerfield** [comment]
Glad you added those terms to the index. Maybe also add patricia tree since sometimes people write that.
Is it me that is supposed to do Close and comment or you? (I'm not much of a github user, at least not the details of it all.)

**Comment by Araq** [comment]
I didn't add these terms. I remarked that you should add them. I have shown how to do that, in the comment use this syntax

```
 `term`:idx:
```

So that they are indexed by our documentation generator.

**Comment by Araq** [comment]
> Is it me that is supposed to do Close and comment or you? 

In general it's me who closes the issue.

**Pull request closed without merging**